backend/cellguide/pipeline/gpt_descriptions/__init__.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from backend.cellguide.pipeline.gpt_descriptions.gpt_description_generator import generate_all_gpt_descriptions
+from backend.cellguide.pipeline.ontology_tree import get_ontology_tree_builder
+from backend.wmg.api.wmg_api_config import WMG_API_SNAPSHOT_SCHEMA_VERSION
+from backend.wmg.data.snapshot import load_snapshot
+
+logging.basicConfig(level=logging.INFO)
+
+
+def run():
```

Comment:
Can you follow the pattern used in the other __init__ files (a `run` function that calls a `get_*` function). The `get_*` function is just a one-line wrapper around your generate function. For example, [source collections](https://github.com/chanzuckerberg/single-cell-data-portal/blob/19aafe00de0787274131fbb78f08e465f1461b47/backend/cellguide/pipeline/source_collections/__init__.py#L21). this pattern ensures consistency so the behavior of the functions are clearly delineated. For example, `run` gets the data and outputs the data. `get` gets the data.

Reply:
This pattern was adopted to make it easier to disentangle data generation from data output while keeping imports less confusing (particularly useful for [the script](https://github.com/chanzuckerberg/single-cell-data-portal/blob/19aafe00de0787274131fbb78f08e465f1461b47/scripts/generate_cellguide_pipeline_test_fixtures.py#L65) used to generate test fixtures).